metricbeat/docs/modules/ceph.asciidoc

```diff
@@ -6,7 +6,10 @@ This file is generated! See scripts/mage/docs_collector.go
 == Ceph module
 
 The Ceph module collects metrics by submitting HTTP GET requests to
-the http://docs.ceph.com/docs/master/man/8/ceph-rest-api/[ceph-rest-api]. The default metricsets are `cluster_disk`, `cluster_health`, `monitor_health`, `pool_disk`, `osd_tree`.
+the https://docs.ceph.com/docs/jewel/man/8/ceph-rest-api/[ceph-rest-api]. The default metricsets are `cluster_disk`, `cluster_health`, `monitor_health`, `pool_disk`, `osd_tree`.
+
+Metricsets connecting to the Ceph REST API uses by default the service exposed on port 5000.
+Metricsets using the Ceph Manager Daemon communicate with the API exposed by default on port 8003 (SSL encryption).
```

Comment:
We could also add a block with an example config for these modules to `config.reference.yml`, maybe with some comments about what block should be added, so this appears as example configuration below. wdyt?

Reply:
Actually we have a dedicated one: `config-mgr.yml`. Do you think we should modify the reference too?

Reply:
I've updated the reference configuration.

Reply:
:+1: thanks